public/controllers/agent/components/register-agent.js

```diff
@@ -379,17 +378,26 @@ export const RegisterAgent = withErrorBoundary(
       }
     }
 
-    render() {
-      const appVersionMajorDotMinor = this.state.wazuhVersion.split('.').slice(0, 2).join('.');
-      const urlCheckConnectionDocumentation = `https://documentation.wazuh.com/${appVersionMajorDotMinor}/user-manual/agents/agent-connection.html`;
-      const textAndLinkToCheckConnectionDocumentation = (
-        <p style={pTextCheckConnectionStyle}>
-          To verify the connection with the Manager, please follow this{' '}
-          <a href={urlCheckConnectionDocumentation} target="_blank">
-            document.
-          </a>
-        </p>
-      );
+  getHighlightCodeLanguage(selectedSO){
+    if(selectedSO.toLowerCase() === 'win'){
+      const iKibanaVersion = parseFloat(this.state.kibanaVersion.split('.').slice(0, 2).join('.'),2);
+      return iKibanaVersion < 7.14 ? 'ps' : 'powershell';
+    }else{
```

Comment:
run prettier here

Reply:
I will applied prettier to the entire document. Thanks you!